lighthouse-core/runner.js

```diff
@@ -212,8 +229,11 @@ class Runner {
         entryType: entry.entryType,
       };
     }).sort((a, b) => a.startTime - b.startTime);
-    const runnerEntry = timingEntries.find(e => e.name === 'lh:runner:run');
-    return {entries: timingEntries, total: runnerEntry?.duration || 0};
+    const gatherEntry = timingEntries.find(e => e.name === 'lh:runner:gather');
+    const auditEntry = timingEntries.find(e => e.name === 'lh:runner:audit');
+    const gatherTiming = gatherEntry?.duration || 0;
+    const auditTiming = auditEntry?.duration || 0;
+    return {entries: timingEntries, total: gatherTiming + auditTiming};
```

Comment:
This might encounter issues in a flow with multiple steps. I don't know if a flow step is guaranteed to query the correct timing entry from the logger.

@brendankenny this might be a reason to use a `Runner` instance instead. We could store timing information on that instance instead of querying the logger.

Reply:
It's not causing any issues right now because `log.takeTimingEntries` clears everything between each flow step. I'll investigate this in the next part. There's probably a way to remember timings for each step without adding state to `Runner`.

Reply:
FWIW it's not clear what `total` should mean going forward. Right now it's the total of the LH run that _just_ completed, so if run with `-A`, the `-G` timings are in the `entries` array, but not included in the total. That would be changed here, and I kind of prefer the old way, but I'm not sure if the `total` timing of `-A` runs is important enough to care (if e.g. you're trying to optimizing something, you're likely looking at some other timing or tool).

In a user flow, is the amount of time to gather and audit a particular step the most meaningful interpretation of "total"? Seems ok.

Should add some total tests if we're making a decision.

Also it's worth pointing out that `total` was a placeholder while we figured out `entries` and only kept it around because it was kind of useful after `entries` landed, so the only bar to clear here is "kind of useful" :)